packages/addons/service/dispmanx_vnc/package.mk

```
@@ -1,35 +1,26 @@
 # SPDX-License-Identifier: GPL-2.0
-# Copyright (C) 2016-present Team LibreELEC (https://libreelec.tv)
+# Copyright (C) 2020-present Team LibreELEC (https://libreelec.tv)
 
 PKG_NAME="dispmanx_vnc"
-PKG_VERSION="5bde54b5a43a2e6f65994f3b9c665db90c45923f"
-PKG_SHA256="1f096cd70d86620596db947757486064fe37dc2766ad33530318c35b279472aa"
-PKG_VERSION_DATE="2018-08-18"
-PKG_REV="106"
+PKG_VERSION="1.0"
+PKG_REV="107"
```

```suggestion
PKG_REV="100"
```

no need for bumping, next repo bump solves that issue (already pred)

---

thanks, fixed that now